test/test.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
-const t = require('tap')
+const t = require('node:test')
+const assert = require('node:assert')
 const test = t.test
```

Comment:
nit import test on same line
```suggestion
const { test } = require('node:test')
```